tests/cypress/integration/5-AI-SiteGen-onboarding-flow/6-preview.cy.js

```diff
@@ -6,12 +6,26 @@ import {
 	LightBGCheck,
 	ProgressBarCheck,
 } from '../wp-module-support/siteGen.cy';
+import {
+	apiList,
+	siteGenMockAll,
+	homePagesMock,
+} from '../wp-module-support/MockApi.cy';
 
 describe( 'SiteGen Site Preview Step', function () {
 	before( () => {
 		cy.visit(
 			'wp-admin/index.php?page=nfd-onboarding#/sitegen/step/preview'
 		);
+		cy.intercept( apiList.sitegen, ( req ) => {
+			siteGenMockAll( req );
+		} ).as( 'sitegenCalls' );
+
+		cy.intercept( apiList.homepages, ( req ) => {
+			homePagesMock( req );
+		} ).as( 'homePageCall' );
+		cy.timeout(120000);
+		cy.wait(5000);
 		cy.wait( 5000 );
```

Comment:
Do we make the calls here? Like I don't think we can navigate to this step without all the meta calls being completed.